From 2079e909b2abd3611b854c62f54fabe14d47d18e Mon Sep 17 00:00:00 2001
From: Shenwei Wang <shenwei.wang@nxp.com>
Date: Thu, 19 Apr 2018 16:06:43 -0500
Subject: [PATCH 3675/5242] MLK-18088: arm: dts: enable the 7ulp mmdc
 profiling feature

commit  407c6a277538d25621c7361d1201c7f9547d7dff from
https://source.codeaurora.org/external/imx/linux-imx.git

7ULP uses the same mmdc profiling block as i.mx6q. Added the
"fsl,imx6q-mmdc" compatible string to enable the mmdc profiling
feature.

Reviewed-by: Frank Li <frank.li@nxp.com>
Signed-off-by: Shenwei Wang <shenwei.wang@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx7ulp.dtsi |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/imx7ulp.dtsi b/arch/arm/boot/dts/imx7ulp.dtsi
index 677b63a..fd5061c 100644
--- a/arch/arm/boot/dts/imx7ulp.dtsi
+++ b/arch/arm/boot/dts/imx7ulp.dtsi
@@ -506,7 +506,7 @@
 		};
 
 		mmdc: mmdc@40ab0000 {
-			compatible = "fsl,imx7ulp-mmdc";
+			compatible = "fsl,imx7ulp-mmdc", "fsl,imx6q-mmdc";
 			reg = <0x40ab0000 0x4000>;
 		};
 
-- 
1.7.9.5

